rst_files/version_control.rst

```diff
@@ -237,11 +244,9 @@ Let's unpack the different components
 
 * The *labels* dropdown lets you tag the issue with labels visible from the issues page, such as "high priority" or "feature request" 
 
-* It's possible to tag other issues and collaborators (including in different repos) by linking to them in the comments. This is part of what's called *GitHub-Flavored Markdown* 
+* It's possible to tag other issues and collaborators (including in different repos) by linking to them in the comments -- this is part of what's called *GitHub-Flavored Markdown* 
 
-For an example of an issue, see `example issue <https://github.com/quanteconuser/example_repository/issues/1>`_ 
-
-The checkbox idiom is a common one to manage projects in GitHub 
```

Comment:
I think we also need to talk about the checkboxes since it's so common

Reply:
Ok, I'd profer to just say 'Checkboxes are useful to manage project tasks'

Reply:
That works.